### DISCUSSION TITLE
Typing indicator and read receipts

### DISCUSSION THREAD

**Discussion by xk4rim** [tweepy/tweepy#1855] [Ideas]
Hi. I could not find any Tweepy functions that handle typing indicators and read receipts endpoints. It would be nice to add them to Tweepy.

https://developer.twitter.com/en/docs/twitter-api/v1/direct-messages/typing-indicator-and-read-receipts/api-reference/new-typing-indicator

**Comment by Harmon758** [comment]
Support for both have been added with https://github.com/tweepy/tweepy/commit/e31e640ed1dcab90bd1ac987c051462568744dda.

**Reply by xk4rim** [reply]
Thank you very much!
